task.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+
+class Task:
+
+## constructor for the task class 
+## @param id the uniqe id of the project
+## @param status the status of the task (pending as deafault)
+
+    def __init__(self, id, status='pending'):
+        self.creation_date = datetime.datetime.now()
+        self.id = id
+        self.status = status
+
+## methods fot the class task. the object of the class task is given as parameter 
+
+def get_creation_date(task):
```

Comment:
instead of this lets create a function that returns all the details of a single task